src/components/controllers/doc-page.jsx

```diff
@@ -20,29 +20,27 @@ export function DocPage() {
 		return <NotFound />;
 	}
 
-	return <DocLayout />;
+	return <DocLayout isGuide />;
 }
 
-export function DocLayout() {
+export function DocLayout({ isGuide = false }) {
 	const { path } = useLocation();
 	const [lang] = useLanguage();
 
 	const { html, meta } = useContent([lang, path === '/' ? 'index' : path]);
 	useTitle(meta.title);
 	useDescription(meta.description);
 
-	const hasSidebar = meta.toc !== false && isDocPage(path);
```

Comment:
Redundant; the home page is the only one with `meta.toc == false`, and the sidebar only appears on `/guide/*` pages. Better to switch it to `isGuide` as that's essentially what we're checking.